cohere.ts

```diff
@@ -138,6 +138,15 @@ class Cohere implements CohereService {
   public classify(
     config: models.classifyRequest
```

Comment:
why is this called config

Reply:
@lfayoux I hate it. but that's the way all (or maybe most) of the SDK functions are implemented, as an equivalent to "kwargs" in python